docs/installingNative.md

```diff
@@ -130,43 +119,48 @@ Now you can install the other dependencies.
   cd PrairieLearn
   ```
 
-!!! note "Setup a venv"
+- Set up a Python virtual environment:
 
-    It is recommended to use a virtual environment for Python dependencies. You can create a virtual environment within PrairieLearn:
-
-    === "Native"
+  === "uv"
 
         ```sh
-        python3.10 -m venv venv
-        source venv/bin/activate
+        uv venv --python 3.10 --seed
+        source .venv/bin/activate
         ```
 
-    === "uv"
+  === "Native"
 
         ```sh
-        uv venv --python 3.10
-        source venv/bin/activate
+        python3.10 -m venv .venv
+        source .venv/bin/activate
         ```
 
+  You can run `deactivate` to exit the virtual environment, and `source .venv/bin/activate` to re-enter it.
+
+- On macOS, set the following environment variables so that `pygraphviz` can [find the necessary headers](https://github.com/pygraphviz/pygraphviz/blob/main/INSTALL.txt):
+
+  ```sh
+  cat <<EOF >> .venv/bin/activate
+  export CFLAGS="-I$(brew --prefix graphviz)/include"
+  export LDFLAGS="-L$(brew --prefix graphviz)/lib"
+  EOF
+  source .venv/bin/activate
```

Comment:
nit: either add a newline here or two code block to indicate that this is doing two different operations: 1 should be run only once, but the source should run every time.

Reply:
Done in 9dec613a82daba80691e05e443c07e609caa0234